tests/fixtures/index.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+/*
+ * This file is part of the Panther project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+echo 'OK';
```

Comment:
What is this file ?

Reply:
Just a file that will return a 200. The Tests suite needs it to be sure the test webserver is started

Reply:
Note: I was not needed before, because the code did not perform any assertion on the status code. Now it does...

Reply:
wouldn't this mean that if your homepage has a bug, starting the server does not work ?

Reply:
I agree we should only check that we get an HTTP response, regardless of the status.